ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -207,6 +212,22 @@ public String getRegion() {
 		return isClustered() ? getClusterProperties().getProperty(PROP_CLUSTER_REGION) : NONE_REGION;
 	}
 
+	/**
+	 * Get the current region and subregion from the configuration.
+	 *
+	 * @return region and subregion. If it's not clustered mode, return "NONE"
+	 */
+	public Map<String, String> getRegionWithSubregion() {

```

Comment:
Why do you return map here?

Reply:
I use [this method](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/config/MemberAttributeConfig.java#L42) to add attributes to hazelcast member.
It is used when clustered hazelcast members share own region information.

Reply:
Hazelcast member attribute is accept only String type.

```
// Map data format

"region": "kor"
"subregion: "sub1,sub2,sub3"
```